src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -218,7 +218,7 @@ const useCloudEnvironmentPolling = (googleProject, workspace) => {
     try {
       const [newDisks, newRuntimes] = !!workspace ? await Promise.all([
         Ajax(signal).Disks.list({ googleProject, creator: getUser().email, includeLabels: 'saturnApplication,saturnWorkspaceName', saturnWorkspaceName }),
-        Ajax(signal).Runtimes.listV2({ creator: getUser().email, saturnWorkspaceNamespace, saturnWorkspaceName })
+        !!googleProject ? Ajax(signal).Runtimes.listV2({ googleProject, creator: getUser().email, saturnWorkspaceNamespace, saturnWorkspaceName }) : Ajax(signal).Runtimes.listV2({ creator: getUser().email, saturnWorkspaceNamespace, saturnWorkspaceName })
```

Comment:
I dont really like how this reads... it uses a side effect to determine which information to filter on when we really want to include those filters if they are defined.

could we instead construct an object that is destructured into the list call with null values filtered?

Something like 
```
const labelFilters = _.omit(_.isUndefined,{ saturnWorkspaceNamespace, saturnWorkspaceName })
Ajax(signal).Runtimes.listV2({ googleProject, creator: getUser().email, ...labelFilters })
```

(note, _.omit code syntax may not be right, but the idea is there)

Reply:
not sure exactly what you mean... 

I think the approach here covers all cases, we simply include all filters that we can find. For azure runtimes, google project isn't defined, so this is handled. For pre-PPW workspaces that share the same google project, we use workspaceName+workspaceNamespace, which represent a unique identifier for a workspace together, so that case is covered as well.